third-party/py/setuptools/BUCK

```diff
@@ -1,17 +1,2 @@
-python_library(
-  name = 'pkg_resources',
-  base_module = '',
-  srcs = [
-    'pkg_resources.py',
-  ],
-  visibility = [
-    'PUBLIC',
-  ],
-)
-
-export_file(
-  name = 'pkg_resources.py',
-  visibility = [
-    'PUBLIC',
-  ],
-)
+# Note: you shouldn't depend on pkg_resources directly,
+# as it's included in every pex by make_pex.
```

Comment:
But the pex builder uses it right?  So the pex-builder binary we build (to build the actual PEXs) should depend on this, no?